### ISSUE TITLE
"check-name" is an invalid argument

### ISSUE THREAD

**Issue by asafch** [DeLaGuardo/clojure-lint-action#11]
I've configured the action in a workflow like so:
```
  - name: Lint
    uses: DeLaGuardo/clojure-lint-action@v1
    with:
      clj-kondo-args: --lint src test --cache false
      check-name: clj-kondo liniting report
      github_token: ${{ secrets.GITHUB_TOKEN }}
```

When the workflow is run I get this warning: `Unexpected input(s) 'check-name', valid inputs are ['entryPoint', 'args', 'clj-kondo-args', 'github_token']`
And indeed the action reports cli-kondo's report with the default name "clj-kondo check" instead of the check name I configured.

**Comment by DeLaGuardo** [comment]
Thanks! I will fix it soon
